### PR TITLE
Add CI checks and setup

### PR DESCRIPTION
**Pull request by Yash-10**
From #9. Needed to modify `setup.cfg`, otherwise yielded build errors.

Few changes in `src` files were made due to the new CI checks, otherwise gave errors.

### CODE REVIEW

**Comment by veds12** [comment]
@Yash-10 can you solve the conflicts arising due to the renaming?

**Comment by Yash-10** [comment]
I saw metadata from `setup.cfg` was transferred to `setup.py`. Needed to add `MANIFEST.in`, else gave errors during `tox` invocation.

File: src/jeta/loss.py [hunk 1]
```diff
@@ -2,8 +2,9 @@
 import jax.numpy as jnp
 import optax
 
+
 @jax.jit
-def task_loss(model, batch):
+def task_loss(model, batch, ways):
```
Comment (veds12) [inline src/jeta/loss.py]: `ways` is unused?

File: src/jeta/opti_trainer.py [hunk 2]
```diff
@@ -47,7 +49,7 @@ def loss(model, train_x, train_y, val_x, val_y):
         # TODO: Allow *args/**kwargs for `fit_task`.
         updated_model = fit_task(model, train_batch)
         # Evaluate on query set to get the loss on the query set.
-        loss = task_loss(updated_model, val_batch)
+        loss = partial(task_loss, ways=ways)(updated_model, val_batch)
```
Comment (veds12) [inline src/jeta/opti_trainer.py]: Same as above

**Review by veds12 — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[options.packages.find]
+where = src
```
Comment (veds12) [inline setup.cfg]: Don't need to include this in config file

File: src/jeta/loss.py (hunk 1 above)
Comment (Yash-10) [reply src/jeta/loss.py]: `ways` is [used](https://github.com/Yash-10/max/blob/22771a32c745a399b88440e062ae1942050b7014/src/jeta/loss.py#L14) below, right?

File: setup.cfg (hunk 3 above)
Comment (Yash-10) [reply setup.cfg]: Removed now :+1:

**Comment by Yash-10** [comment]
@veds12 I think the only other suggestion was to remove `src/jeta` and instead keep it as `jeta`, right? If yes, I think this is ready for a final review...

**Comment by veds12** [comment]
Yes, thanks. I have been occupied this week. I'll try to review and merge
tomorrow.
